docs/Config_Reference.md

```diff
@@ -1128,7 +1128,43 @@ the nature of skew correction these lengths are set via gcode. See
 [skew_correction]
 ```
 
-## Customized homing
+## [frame_expansion_compensation]
+
+Compensate for vertical toolhead movement caused by thermal expansion of the
+frame in real-time using a temperature probe coupled to a vertical member.
+E.g. A glass bead thermistor suspended halfway down the central channel of an
+2020 aluminum extrusion.
+
+See also: [extended g-code commands](G-Codes.md#Frame-Expansion-Compensation).
+
+```
+[frame_expansion_compensation]
+#temp_coeff:
+#   The temperature coefficient of expansion, in mm/K. For example, a
+#   temp_coeff of 0.01 mm/K will move the Z axis downwards by 0.01 mm for every
+#   Kelvin/degree celcius that the frame temperature increases. Defaults to 0.0,
+#   no offset.
+#temp_sensor:
+#   Temperature sensor to use for frame temp measurement. Use full config
+#   section name without quoutes. E.g. temperature_sensor frame
+#smooth_time:
+#   Smoothing window applied to the temp_sensor, in seconds. Can reduce motor
+#   noise from excessive small corrections in response to sensor noise. The
+#   default is 2.0 seconds.
+#max_comp_z:
+#   Disables compensation above this Z height [mm]. The last computed correction
+#   will remain applied until the toolhead moves below the specified Z position
+#   again. The default is 0.0mm (always on).
```

Comment:
If it's disabled above the specified Z, with "0.0" it should be always "off", not always "on".

Reply:
Good point, will switch to the 99999 default "always on" for clarity.

Reply:
Then better write "The default is 999999 mm (when enabled, compensation active at every Z value).".
To avoid confusion between "temp_coeff" which states "enabled when >0", and this one which states "always enabled".